dockerfiles/Dockerfile_alpine

```diff
@@ -0,0 +1,8 @@
+FROM --platform=i386 i386/alpine
+RUN apk update && apk upgrade
+RUN apk add linux-virt
+RUN apk add openrc
+
+# We get all dependencies to setup-alpine
+RUN apk add --no-cache linux-virt gcc clang musl-dev make openrc icu-data-full curl git
```

Comment:
I see `openrc` is installed twice. Is this required?

Also, is there a reason for installing in 3 separate steps and using `--no-cache`?